azext_iot/tests/iothub/certificate/test_iothub_certificate_root_authority_int.py

```diff
@@ -0,0 +1,67 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.tests.iothub import IoTLiveScenarioTest
+from azext_iot.tests.iothub import DATAPLANE_AUTH_TYPES
+
+
+class TestIotHubCertificateRoot(IoTLiveScenarioTest):
+    def __init__(self, test_case):
+        super(TestIotHubCertificateRoot, self).__init__(test_case)
+
+    def test_certificate_root(self):
+        for auth_phase in DATAPLANE_AUTH_TYPES:
```

Comment:
There are no connection string or keys at play for this control plane only functionality.